src/NBthDEOptPara.cpp

```diff
@@ -1,6 +1,21 @@
+
 #include <cmath>
 #include "GeoDiff.h"
+#include <Rcpp.h>
 #include <RcppArmadillo.h>
+#include <chrono>
+#include <iostream>
+
+using namespace std::chrono;
+
+
+// written by Lei Yang (lyang@nanostring.com) and Ned Booker (ned@procogia.com)
```

Comment:
I would remove this section and stick to adding your name to the authors list in the description file